src/client/package.json

```diff
@@ -63,6 +65,7 @@
     "lodash": "^4.17.21",
     "openfin-adapter": "^27.71.21",
     "openfin-notifications": "^1.10.1",
+    "playwright": "^1.29.1",
```

Comment:
should be dev dep .. unless it is because we need in CI (as we do `npm ci` there before running e2e)
in which case, I would prefer that we did full npm install in CI, rather than label this a production dependency